### PR TITLE
K8SPS-168: Check if the specified CM issuer exist

### PR DESCRIPTION
**Pull request by inelpandzic**
[K8SPS-168](https://jira.percona.com/browse/K8SPS-168) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

---
**Problem:**
When the wrong `spec.tls.issuerConf` is specified, we don't provide a clear error message.

**Solution:**
Besides having a more clear error message, when we get issuerConf from the user, first check if the issuer is created and ready before proceeding.

**CHECKLIST**
---
**Jira**
- [x] Is the Jira ticket created and referenced properly?
- [x] Does the Jira ticket have the proper statuses for documentation (`Needs Doc`) and QA (`Needs QA`)?
- [x] Does the Jira ticket link to the proper milestone (Fix Version field)?

**Tests**
- [x] Is an E2E test/test case added for the new feature/change?
- [x] Are unit tests added where appropriate?

**Config/Logging/Testability**
- [x] Are all needed new/changed options added to default YAML files?
- [x] Are the manifests (crd/bundle) regenerated if needed?
- [x] Did we add proper logging messages for operator actions?
- [x] Did we ensure compatibility with the previous version or cluster upgrade process?
- [x] Does the change support oldest and newest supported PS version?
- [x] Does the change support oldest and newest supported Kubernetes version?

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/gr-demand-backup.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/gr-scaling.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/gr-tls-cert-manager.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/one-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/sidecars.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/e68e7f0/version-service.log) |
| We run 22 out of 22|

commit: https://github.com/percona/percona-server-mysql-operator/pull/317/commits/e68e7f07f44d3ff518cc8fabe71c619703b37bcf
image: `perconalab/percona-server-mysql-operator:PR-317-e68e7f0`

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/demand-backup.log) |
| gr-demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-demand-backup.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-scaling.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-tls-cert-manager.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/one-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/sidecars.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/version-service.log) |
| We run 22 out of 22|

commit: https://github.com/percona/percona-server-mysql-operator/pull/317/commits/3de5c8fab4f1fad7dc711b8f1d87465588204e8d
image: `perconalab/percona-server-mysql-operator:PR-317-3de5c8f`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/demand-backup.log) |
| gr-demand-backup | [failure](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-demand-backup.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-scaling.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-tls-cert-manager.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/one-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/sidecars.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/version-service.log) |
| We run 22 out of 22|

commit: https://github.com/percona/percona-server-mysql-operator/pull/317/commits/3de5c8fab4f1fad7dc711b8f1d87465588204e8d
image: `perconalab/percona-server-mysql-operator:PR-317-3de5c8f`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-demand-backup.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-scaling.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/gr-tls-cert-manager.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/one-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/sidecars.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/3de5c8f/version-service.log) |
| We run 22 out of 22|

commit: https://github.com/percona/percona-server-mysql-operator/pull/317/commits/3de5c8fab4f1fad7dc711b8f1d87465588204e8d
image: `perconalab/percona-server-mysql-operator:PR-317-3de5c8f`

File: pkg/controller/ps/controller.go [hunk 1]
```diff
@@ -1552,6 +1552,17 @@ func (r *PerconaServerMySQLReconciler) createSSLByCertManager(ctx context.Contex
...
 		issuerName = cr.Spec.TLS.IssuerConf.Name
 		issuerGroup = cr.Spec.TLS.IssuerConf.Group
+
+		isr := &cm.Issuer{}
+		err := r.Get(ctx, types.NamespacedName{
+			Namespace: cr.Namespace,
+			Name:      issuerName,
+		}, isr)
+		if k8serrors.IsNotFound(err) {
+			log.FromContext(ctx).Error(err, "Can't find specified cert-manager issuer", "issuer", issuerName)
```
Comment (egegunes) [inline pkg/controller/ps/controller.go:1562]: do we need a separate log? if we wrap the error with this message, it'll be logged anyway

Comment (inelpandzic) [reply pkg/controller/ps/controller.go:1562]: @egegunes I was waiting for that question :) Yes, that is totally true, my idea was to make it even more obvious to the user (so does not have to go through the generic `reconcile` error, but I also think we don't need it necessarily.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/gr-demand-backup.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/gr-scaling.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/gr-tls-cert-manager.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/one-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/sidecars.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/ff269bb/version-service.log) |
| We run 22 out of 22|

commit: https://github.com/percona/percona-server-mysql-operator/pull/317/commits/ff269bb196af07e8f3f47a196dd2b7916abecd25
image: `perconalab/percona-server-mysql-operator:PR-317-ff269bb`

File: pkg/controller/ps/controller.go (hunk 1 above)
Comment (inelpandzic) [reply pkg/controller/ps/controller.go]: Removed

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/async-ignore-annotations.log) |
| auto-config | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/auto-config.log) |
| config | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/config.log) |
| config-router | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/config-router.log) |
| demand-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/demand-backup.log) |
| gr-demand-backup | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/gr-demand-backup.log) |
| gr-ignore-annotations | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/gr-ignore-annotations.log) |
| gr-init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/gr-init-deploy.log) |
| gr-scaling | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/gr-scaling.log) |
| gr-tls-cert-manager | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/gr-tls-cert-manager.log) |
| haproxy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/haproxy.log) |
| init-deploy | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/init-deploy.log) |
| limits | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/limits.log) |
| monitoring | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/monitoring.log) |
| one-pod | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/one-pod.log) |
| scaling | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/scaling.log) |
| semi-sync | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/semi-sync.log) |
| service-per-pod | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/service-per-pod.log) |
| sidecars | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/sidecars.log) |
| tls-cert-manager | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/tls-cert-manager.log) |
| users | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/users.log) |
| version-service | [skipped](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/d3dedcd/version-service.log) |
| We run 0 out of 22|

commit: https://github.com/percona/percona-server-mysql-operator/pull/317/commits/d3dedcde5c764bdbedee9a6a423ea8dc763b6988
image: `perconalab/percona-server-mysql-operator:PR-317-d3dedcd`

**Review by egegunes — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| async-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/async-ignore-annotations.log) |
| auto-config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/auto-config.log) |
| config | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/config.log) |
| config-router | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/config-router.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/demand-backup.log) |
| gr-demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/gr-demand-backup.log) |
| gr-ignore-annotations | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/gr-ignore-annotations.log) |
| gr-init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/gr-init-deploy.log) |
| gr-scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/gr-scaling.log) |
| gr-tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/gr-tls-cert-manager.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/init-deploy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/limits.log) |
| monitoring | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/monitoring.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/one-pod.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/scaling.log) |
| semi-sync | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/semi-sync.log) |
| service-per-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/service-per-pod.log) |
| sidecars | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/sidecars.log) |
| tls-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/tls-cert-manager.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/users.log) |
| version-service | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-ps-operator/PR-317/605c2a8/version-service.log) |
| We run 22 out of 22|

commit: https://github.com/percona/percona-server-mysql-operator/pull/317/commits/605c2a8296937239fb6173749aa56caeed370e91
image: `perconalab/percona-server-mysql-operator:PR-317-605c2a8`

**Pull request merged**